src/main/solution_space/distance.py

```diff
@@ -1,44 +1,88 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
```

Comment:
Please, write **Todo comment** with my comments in the file, we will fix it in the future because now we don't use distance